icepyx/core/visualization.py

```diff
@@ -466,7 +466,7 @@ def parallel_request_OA(self) -> da.array:
             OA_data_da = da.concatenate(requested_OA_data, axis=0)
             return OA_data_da
 
-    def viz_elevation(self) -> (hv.DynamicMap, hv.Layout):
+    def viz_elevation(self) -> tuple[hv.DynamicMap, hv.Layout]:
```

Comment:
Aren't these the same because `()` is by definition a tuple? Or is there something internally that happens differently?

Reply:
These are actually different; note the square brackets instead of parenthesis. `(a, b)` is equivalent to `tuple([a, b])`; these both create a runtime tuple object. When writing type annotations, the brackets indicate "arguments" to the tuple _type_. Things on the right-hand side of the `->` are always types and don't affect the runtime behavior of the function (though they _can_ be read as metadata intentionally at runtime!), and same for things on the right-hand side of the `:` in function argument signatures.

What we're saying here is that the function returns an object of type `tuple` with two elements: first element is an object of type `hv.DynamicMap`, and the second is an object of type `hv.Layout`.